src/styles/main.scss

```diff
@@ -1,3 +1,58 @@
+@import "./variables";
+
 body {
   margin: 0;
+  font-family: Arial, Helvetica, sans-serif;
+  font-size: 30px;
+}
+
+.container {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  display: flex;
+  flex-wrap: wrap;
+  gap: $calendar-day-gap;
+  width: 7 * $calendar-day-width + 16px;
+
+  &__day {
+    box-sizing: border-box;
+    height: $calendar-day-width;
+    width: $calendar-day-width;
+    border: 1px solid black;
+    background-color: #eee;
+    transition: transform 0.3s;
+
+    &:hover {
+      cursor: pointer;
+      transform: translateY(-20px);
+      background-color: pink;
+    }
+
+    @for $index from 1 through $the-last-day {
+      &:nth-child(#{$index})::before {
+        content: "#{$index}";
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        height: 100%;
+      }
+    }
```

Comment:
7. [SASS] - Don't use SASS loops for styles that stay the same for all elements of the group, e.g. display or position.

Reply:
Fix student comment. Checklist #7